### DISCUSSION TITLE
Error while trying to use the "Can we go somewhere else?" command

### DISCUSSION THREAD

**Discussion by Gwen-the-pen** [Monika-After-Story/MonikaModDev#9386] [QA - tech support + general]
![image](https://user-images.githubusercontent.com/109734782/180637205-70013a68-2adb-4106-a606-046c4c204a8d.png)
I cannot even use MAS anymore. How do I fix this?? I do have "MJ' S selector city", and the "Extra Everything"(i think) mod if that helps at all

**Comment by Booplicate** [comment]
You're missing some images for that submods, try to reinstall.
